api/server.go

```diff
@@ -1209,16 +1228,21 @@ func (s *server) addressesAddrOutputsSFHandler(jc jape.Context) {
 		return
 	}
 
+	var useTPool bool
```

Comment:
Ngl this is a bit confusing to me. Had to go back and forth between the API and manager methods. Cause `useTPool` and `excludePool` doesn't sounds like it would do the same thing to me. Cause it sounds like "please use outputs from the tpool as well" to me rather than "use the tpool to exclude outputs".